src/applications/vaos/services/patient/index.js

```diff
@@ -229,6 +229,7 @@ function logEligibilityExplanation(
  * @param {boolean} [params.useFeSourceOfTruth=false] whether to use vets-api payload as the FE source of truth
  * @param {boolean} [params.useFeSourceOfTruthCC=false] whether to use vets-api payload as the FE source of truth for CC appointments and requests
  * @param {boolean} [params.useFeSourceOfTruthVA=false] whether to use vets-api payload as the FE source of truth for VA appointments and requests
+ * @param {boolean} [params.usePastVisitMHFilter=false] whether to use vets-api payload as the FE source of truth for VA appointments and requests
```

Comment:
nit: doc comment seems inaccurate

Reply:
Yep, will fix. I thought I fixed that. Must have missed it when I reverted something.

Reply:
Fixed